yarn.lock

```diff
@@ -437,12 +449,14 @@ caseless@~0.12.0:
   version "0.12.0"
   resolved "https://registry.yarnpkg.com/caseless/-/caseless-0.12.0.tgz#1b681c21ff84033c826543090689420d187151dc"
 
-ccxt@1.10.641:
-  version "1.10.641"
-  resolved "https://registry.yarnpkg.com/ccxt/-/ccxt-1.10.641.tgz#ad6d104be952b48526219baed633876d1423244d"
+ccxt@1.12.118:
+  version "1.12.118"
+  resolved "https://npm.orcaware.com/ccxt/-/ccxt-1.12.118.tgz#2c82b70e171a08c7b3a2cd61dd2b7f469a1fbb2d"
```

Comment:
This was resolved to your private npm, @blair.

Reply:
Thanks for catching that. I've fixed this, updated ccxt to the now current latest and force pushed the branch. PTAL.